src/components/bulk-email-tool/bulk-email-form/BulkEmailRecepient.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
```

Comment:
Nit: Just calling out that `Recipient` is misspelled in the JSX file name.

Reply:
I did end up renaming the file to the correct spelling